data/human/culture conversations.txt

```diff
@@ -120,6 +120,37 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Local Politics"
+	minor
+	source
+		government "Free Worlds"
+		attributes "frontier"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`As you land on the pad, you note a general lack of presence around the spaceport. No one scrambled to hook up your ship nor were there a full contingent waiting to receive you. Instead, when you leave your ship, all you find is a note within the spaceport detailing where their overlords can find their yearly tribute.`
```

Comment:
The mixed tense is kinda clunky.
```suggestion
			`As you land on the pad, you note that no one has scrambled to hook up your ship, nor has anybody else acknowledged your arrival. Instead, when you leave your ship, all you find is a note within the spaceport detailing where their overlords can find their yearly tribute.`
```

Reply:
Can we re-try this one retaining the "general lack of presence"?

Reply:
```suggestion
			`As you land on the pad, you note a general lack of presence around the spaceport. No one has scrambled to hook up your ship, nor has anybody else acknowledged your arrival. Instead, when you leave your ship, all you find is a note within the spaceport detailing where the "overlords" can find their yearly tribute.`
```
This look right? I also made a quick change at the bottom - it's not quite obvious who your `their` refers to.

Reply:
I agree to the changes baring the "their," personally. It sounds right that it's "their" overlords over "the" overlords.
Then again, I'm not the one with Merge powers, so someone please give a second opinion.

Reply:
```suggestion
			`As you land on the pad, you note a general lack of presence around the spaceport. No one has scrambled to hook up your ship, nor has anybody else acknowledged your arrival. Instead, when you leave your ship, all you find is a note within the spaceport detailing where their "overlords" can find their yearly tribute.`
```

Reply:
Still not huge on the `their` vs `the`, but I'm cool with letting it slide